packages/rs-dpp/src/document/action_type.rs

```diff
@@ -1,6 +0,0 @@
-pub enum DocumentActionType {
```

Comment:
Did you make sure these weren't being used anywhere?

Reply:
Yes